geoh5py/ui_json/utils.py

```diff
@@ -341,7 +345,11 @@ def monitored_directory_copy(
 
     with fetch_active_workspace(entity.workspace, mode="r"):
         with Workspace.create(working_path / temp_geoh5) as w_s:
-            entity.copy(parent=w_s, copy_children=copy_children)
+            entity.copy(
+                parent=w_s,
+                copy_children=copy_children,
+                copy_property_groups=copy_property_groups,
```

Comment:
Changing here will break the copy for all other entities that don't have the copy_property_groups, or don't 
have property_groups. Let's not do that now and focus on fixing the grid2d instead.